config/cable.yml

```diff
@@ -0,0 +1,10 @@
+development:
```

Comment:
Whats this file about?

Reply:
This is a config file for the Action Cable where we can specify an adapter for each Rails environment.